pkg/apis/sources/v1alpha1/containersource_types.go

```diff
@@ -49,6 +49,11 @@ type ContainerSourceSpec struct {
 	// +patchStrategy=merge
 	Env []corev1.EnvVar `json:"env,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 
+	// ServiceAccountName is the name of the ServiceAccount to use to run this
+	// source.
+	// +optional
+	ServiceAccountName string `json:"serviceAccountName,omitempty"`
```

Comment:
since this is optional, if it's left out, does the right thing happen?

Reply:
Yes. The empty ServiceAccountName value passes down to the Pod via the Deployment's PodSpec and is set to `default` by the apiserver.